packages/core/test/unit/dialects/abstract/query.test.js

```diff
@@ -75,6 +79,7 @@ describe('[ABSTRACT]', () => {
           'players.created': new Date('2017-03-06T15:47:30.000Z'),
           'players.lastModified': new Date('2017-08-22T11:16:44.000Z'),
           'agents.uuid': agentTwoUuid,
+          name: 'joe',
           'agents.id': 'z',
           'agents.name': 'Two',
         },
```

Comment:
I can't comment on the lines below but it might be good to add an expectation for result[0].name to be sure

Reply:
Done